website/blog/2024-12-16-dso-toolkit-67.2.0.mdx

```diff
@@ -0,0 +1,32 @@
+---
+authors: [hansgrimm]
+---
+
+# DSO Toolkit v67.2.0 👹
```

Comment:
Deze is voor jou @hansgrimm. Ik had 🔫  al op de Next milestone gezet, maar per direct aangepast naar 👹.

En het feit dat de release **BREAKING** betkent een nieuwe majeure versie, dus `68.0.0`

Reply:
Hij is niet meer breaking. Die is eraf.